pkg/data/account.go

```diff
@@ -46,38 +43,8 @@ func (a *Account) WriteAccount(account *model.Account, prevLastModifiedOn *int64
 		}
 	}
 
-	putMap, _ := dynamodbattribute.Marshal(account)
-	_, err = a.DynamoDB.PutItem(
-		&dynamodb.PutItemInput{
-			// Query in Lease Table
-			TableName: aws.String(a.TableName),
-			// Find Account for the requested accountId
-			Item: putMap.M,
-			// Condition Expression
-			ConditionExpression:       expr.Condition(),
-			ExpressionAttributeNames:  expr.Names(),
-			ExpressionAttributeValues: expr.Values(),
-			// Return the updated record
-			ReturnValues: aws.String(returnValue),
-		},
-	)
-	var awsErr awserr.Error
-	if gErrors.As(err, &awsErr) {
-		if awsErr.Code() == "ConditionalCheckFailedException" {
-			return errors.NewConflict(
-				"account",
-				*account.ID,
-				fmt.Errorf("unable to update account: accounts has been modified since request was made"))
-		}
-	}
-	if err != nil {
```

Comment:
This part looks like it was missed in the new function.  We still want the other errors captured and returned.

Reply:
Added